AdyenComponents/Apple Pay/ApplePayComponentExtensions.swift

```diff
@@ -113,10 +95,24 @@ extension ApplePayComponent {
             paymentRequest.paymentSummaryItems = summaryItems
             paymentRequest.requiredBillingContactFields = requiredBillingContactFields
             paymentRequest.requiredShippingContactFields = requiredShippingContactFields
+            paymentRequest.billingContact = billingContact
             return paymentRequest
         }
     }
 
+}
+
+extension ApplePayPaymentMethod {
+
+    internal var supportedNetworks: [PKPaymentNetwork] {
+        var networks = ApplePayPaymentMethod.defaultNetworks
+        if let brands = brands {
+            let brandsSet = Set(brands)
+            networks = networks.filter { brandsSet.contains($0.adyenName) }
+        }
+        return networks
+    }
```

Comment:
I thought we agreed to keep the `supportedNetworks` and `defaultNetworks` in the configuration object, right?